pmd-plsql/etc/grammar/PLSQL.jjt

```diff
@@ -3821,7 +3821,7 @@ ASTArgument Argument() :
 {
 	//[LOOKAHEAD(2) UnqualifiedID() "=>"] Expression()
 	[LOOKAHEAD(2) (simpleNode = UnqualifiedID())  ("=>" | <AS> )  ]
-        (Expression() )
+        (Expression() [ <_DEFAULT> Expression() <ON> <CONVERSION> "ERROR" ] )
```

Comment:
I see, this "DEFAULT" clause is part of TO_NUMBER -> https://docs.oracle.com/en/database/oracle/oracle-database/23/sqlrf/TO_NUMBER.html#GUID-D4807212-AFD7-48A7-9AED-BEC3E8809866

```suggestion
        (Expression()
          // e.g. TO_NUMBER allows a DEFAULT clause within an argument
          // https://docs.oracle.com/en/database/oracle/oracle-database/23/sqlrf/TO_NUMBER.html#GUID-D4807212-AFD7-48A7-9AED-BEC3E8809866
          [ <_DEFAULT> Expression() <ON> "CONVERSION" "ERROR" ]
        )
```